src/vs/workbench/contrib/experiments/test/electron-browser/experimentService.test.ts

```diff
@@ -86,11 +86,11 @@ suite('Experiment Service', () => {
 		testConfigurationService = new TestConfigurationService();
 		instantiationService.stub(IConfigurationService, testConfigurationService);
 		instantiationService.stub(ILifecycleService, new TestLifecycleService());
-		instantiationService.stub(IStorageService, { get: (a, b, c) => c, getBoolean: (a, b, c) => c, store: () => { }, remove: () => { } });
+		instantiationService.stub(IStorageService, <Partial<IStorageService>>{ get: (a: string, b: StorageScope, c?: string) => c, getBoolean: (a: string, b: StorageScope, c?: boolean) => c, store: () => { }, remove: () => { } });
```

Comment:
The casts are required as far as I can tell because `IStorageService.get` is overloaded. I can't figure out how to correctly write the typings for overloaded methods so they work with the inferred `Partial<IStorageService>` type. The cast suppresses this (but will still catch incorrect typings)